src/tests/xeon_gold_6252.rs

```diff
@@ -868,6 +868,30 @@ fn extended_topology_info() {
     assert_eq!(t.x2apic_id(), 199); // different from doc, unpinned execution
 }
 
+#[test]
+fn extended_topology_info_v2() {
+    use crate::TopologyType;
+
+    let cpuid = CpuId::with_cpuid_fn(cpuid_reader);
+    let mut e = cpuid
+        .get_extended_topology_info_v2()
+        .expect("Leaf is supported");
+
+    let t = e.next().expect("Have level 0");
+    assert_eq!(t.x2apic_id(), 199); // different from doc, unpinned execution
```

Comment:
I don't understand the comment (both what's different and what unpinned means here)?

Reply:
I'm really not sure—I coped the impl for the existing `extended_topology_info()` test, and only changed the CPUID call used within. Given that it seems like this test is pointless for this CPU (and the Ryzen Matisse), I'll just remove the test entirely.